packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -73,7 +73,7 @@ $ds-c-inset-border-width: $spacer-half;
 
       &::before {
         background-color: $color-gray-lighter;
-        border: 1px solid $color-gray-light;
+        border: $choice-border-width solid $color-gray-light;
```

Comment:
I'm unsure about the `border-width` change here mainly because we are changing the look of the disabled check/radio from 1px to 2px without consulting other teams on this change/update and am wondering how that would impact or surprise them downstream.